openmls/src/test_utils/test_framework/mod.rs

```diff
@@ -81,6 +83,9 @@ pub struct ManagedTestSetup {
     // This maps key package hashes to client ids.
     pub waiting_for_welcome: RefCell<HashMap<Vec<u8>, Vec<u8>>>,
     pub default_mgc: ManagedGroupConfig,
+    /// Flag to indicate if messages should be serialized and de-serialized as
+    /// part of message distribution
+    pub use_codec: bool,
```

Comment:
Can you make this an enum so that we don't have to use `true` and `false` when setting up the config?